Source/FieldSolver/SpectralSolver/SpectralHankelTransform/HankelTransform.cpp

```diff
@@ -169,6 +171,15 @@ HankelTransform::HankelTransform (int const hankel_order,
 
     }
 
+    m_kr.resize(kr.size());
+    m_invM.resize(invM.size());
+    if (!M.empty()) m_M.resize(M.size());
```

Comment:
The check of M.empty() should not be needed since M will always be allocated.

Reply:
OK.  Thanks for the comment.  I will remove the check.